### DISCUSSION TITLE
Trying to backup a large amount of photos and videos from iphone and running into issues

### DISCUSSION THREAD

**Discussion by alexlyee** [immich-app/immich#3225] [Q&A]
In total there's about 330GB of photos and videos I'm trying to import. It was suggested to me to use the CLI to import all of my old photos and videos (that are not on my iphone rn but once were) but many of the things were not imported correctly. There were lots of errors, which might have all been [pertaining to converting the 4k60 videos](https://github.com/immich-app/immich/issues/2271), but either way I know many things are missing and also Immich is having trouble displaying many videos that were uploaded in that same iphone video format directly from my phone. I have no easy way of knowing which files aren't working properly and so I've got to keep local duplicates of all of my photos. 

My phone's out of storage again so I have to delete a bunch of photos. I'm about to do it after using a mac to backup the photos in their native format as well as metadata in neighboring .xml files and I'm wondering-- 
1. will there be a way to import those photos alongside their .xml files directly to immich if I wanted to try importing again, probably because I wanted to try to get all of my files to successfully add (particularly those videos)? If not, I figure I'd have to do some hacking at those files to get them in a format that immich is happy with; what formats does immich have the least problems with?
2. It's my understanding that I do try uploading again it won't create duplicates because the hashes on the files _should_ be the same. But if I had to convert them, or still otherwise they're not, wouldn't I have a boatload of duplicates of the 140GB+ of photos and videos from my phone that are slightly lower quality as they were converted to another format? How do I avoid this without going through and manually trying to delete every photo I think was from my phone?

oh and also, why is my uploads folder 24GB large and it hasn't been modified in a week?

**Comment by alextran1502** [comment]
Can you show the error message?

Can you also confirm that all jobs have finished processing in the job panel on the Administration page on the web?

**Comment by konradr25** [comment]
ad.2 Are you sure that you exported full resolution on mac?
